### DISCUSSION TITLE
Offering some help

### DISCUSSION THREAD

**Discussion by Ayeven** [kookehs/bdo-marketplace#20] [Ideas]
So, I've been using your tool (I prefer the old one where only need items.json) than now. So I would like to offer some help on getting the item id and name from bdocodex. The issue now is I don't know how Golang programming, and the script I write is in Nodejs. I've made a replit for it to demonstrate how it work. [Replit Link](https://repl.it/@Ayeven/bdocodexscrap#README.md).
Here some sample I ran using query "Strawberry". The other parameters of course can be ommited
`[
  {
    "id": "7304",
    "type": "Item",
    "name": "Strawberry",
    "link": "https://bdocodex.com/us/item/7304/"
  },
  {
    "id": "47868",
    "type": "Item",
    "name": "Strawberry Cake",
    "link": "https://bdocodex.com/us/item/47868/"
  },
  {
    "id": "6704",
    "type": "Item",
    "name": "Strawberry Seed",
    "link": "https://bdocodex.com/us/item/6704/"
  },
  {
    "id": "7404",
    "type": "Item",
    "name": "Strawberry Crate",
    "link": "https://bdocodex.com/us/item/7404/"
  },
  {
    "id": "24832",
    "type": "Item",
    "name": "Strawberry Basket",
    "link": "https://bdocodex.com/us/item/24832/"
  }
]`

**Comment by kookehs** [comment]
My bad! `items.json` was removed by accident and has been added again.
If this is in regards to the `config.json` it supports both single and multiple input / output files.
The `master` branch is still under works, but the old version still exists under [releases](https://github.com/kookehs/bdo-marketplace/releases) and its [branch](https://github.com/kookehs/bdo-marketplace/tree/release-branch.0.0.6).

Thanks for the code. I'll take a look. Getting the item IDs automatically would indeed be nice and been discussed a bit before.
There are tools like [bdo-scraper](https://github.com/marceloclp/bdo-scraper) which exists, so I'm not too concerned at this point due to my time constraints.

**Reply by Ayeven** [reply]
the bdo-scraper you link have been abandoned. It still working, but wont be up to date since it using BDODatabase. There is newer one made by same author named "CalphoenJS", but seem having little issue on it end as of now. I tried using it, but run into problem missing modules.
